_maps/Vonneumann.json

```diff
@@ -0,0 +1,34 @@
+{
+	"map_name": "SGV Von Neumann Mark III",
+	"map_link": "neumann",
+    "map_path": "map_files/Von_Neumann",
+    "map_file": ["VNmk3.dmm"],
+	"traits": [{"Linkage": "Self"}],
+	"space_ruin_levels": -1,
+	"space_empty_levels": 0,
+	"shuttles": {
+		"cargo": "cargo_atlas",
+		"ferry": "ferry_fancy"},
+	"mine_disable": 1,
+	"ship_type": "/obj/structure/overmap/nanotrasen/solgov/vnc",
+	"faction": "solgov",
+	"starmap_path": "config/starmap/starmap.json",
+
+    "map_description": "The Von Neumann Mark III is an Experimental Destroyer designed to mount untested technology, carry it into battle, and do so with speed and power. no two ships of the class are similar, and they undergo constant refits to mount new technology slated to be tested in the heat of battle, and often used by SolGov special forces as a light escort and transport. It's fairly common for these ships to spontaniously explode or go missing during their missions due to their questionable outfitting and extremely light armor. the hull structure itself is robust, but that provides little comfort to crew when Enemy fire is tearing through the hull. SolGov has recently started contracting Whiterapids Crews to man these ships instead of providing their own. this model is equipped with a hybrid load of weapons, and a large number of forcefield emitters, as well as a high tech singularity drive that the ship sits on top of.",
+    "manufacturer": "[Redacted]",
+	"mapper": "Truncated-Cake",
+    "commission_date": "May 19, 2264",
+    "strengths": ["Legal energy shields", "Energy weapons", "Compact and easy to repair", "High tech,low maintenance singularity engine", "Highly Manuverable", "Advanced technology", "Smallest ship to ever mount a Turing Drive" ],
+    "weaknesses": ["Fragile", "Power-hungry", "Near Non-existent armor", "Higher crew requirements", "Extremely small" ],
+	"equipment": [
+		"SolGov shield generator",
+		"1 NT-ST049 'Sturm' coaxial railgun",
+		"1 burst phaser MK2",
+		"Triple Laser Anti Missile System",
+		"Defunct Missile Array",
+		"2 light fighters",
+		"2 heavy fighters",
```

Comment:
This fighter complement, in general, is very heavy for a ship of the intended size. In addition. Heavy fighters are uncommon by design.